modules/declarative/pages/policy.adoc

```diff
@@ -0,0 +1,317 @@
+[id="creating-policies"] 
+== Creating policies for the {policy-system}
+
+Policies allow you control communication across the {service-network} from a cluster.
+
+
+.Prerequisites
+
+* Access to a cluster using a `cluster-admin` account
+* The {policy-system} CRD is installed on the cluster
+
+.Procedure
+
+NOTE: Typically, you create a policy CR that combines many elements from the steps below, for example, see xref:about-skupper-policies[]
+
+. xref:allowIncomingLinks[]
+. xref:allowedOutgoingLinksHostnames[]
+. xref:allowedServices[]
+. xref:allowedExposedResources[]
+
+// Type: procedure
+[id="allowIncomingLinks"] 
+=== Implement a policy to allow incoming links
+
+Use `allowIncomingLinks` to enable users create tokens and configure incoming links.
+
+.Procedure
+
+. Determine which namespaces you want to apply this policy to.
+. Create a CR with `allowIncomingLinks` set to `true` or `false`.
+. Create and apply the CR.
+
+For example, the following CR disallows incoming links for all namespaces: 
+[source,yaml]
+----
+apiVersion: skupper.io/v1alpha1
+kind: SkupperClusterPolicy
+metadata:
+  name: allowIncomingLinks
+spec:
+  namespaces:
+    - "*"
+  allowIncomingLinks: true
+----
+
+
+
+
+// Type: procedure
+[id="allowedOutgoingLinksHostnames"] 
+=== Implement a policy to allow outgoing links to specific hosts
+
+Use `allowedOutgoingLinksHostnames` to specify hosts that users can create links to.
+
+. Determine which namespaces you want to apply this policy to.
+. Create a CR with `allowedOutgoingLinksHostnames` set to a pattern of allowed hosts.
+. Create and apply the CR.
+
+For example, the following CR allows links to all subdomains of `example.com` for all namespaces: 
+[source,yaml]
+----
+apiVersion: skupper.io/v1alpha1
+kind: SkupperClusterPolicy
+metadata:
+  name: allowedOutgoingLinksHostnames
+spec:
+  namespaces:
+    - "*"
+  allowedOutgoingLinksHostnames: ['.*\\.example\\.com']
+----
+
+
+// Type: procedure
+[id="allowedServices"] 
+=== Implement a policy to allow specific services
+
+Use `allowedServices` to specify which services can be created or used on the {service-network}.
+
+.Procedure
+
+. Determine which namespaces you want to apply this policy to.
+. Create a CR with `allowedServices` set to specify the services allowed on the {service-network}.
+. Create and apply the CR.
+
+For example, the following CR allows users expose services with the prefix `backend-` for all namespaces: 
+[source,yaml]
+----
+apiVersion: skupper.io/v1alpha1
+kind: SkupperClusterPolicy
+metadata:
+  name: allowedServices
+spec:
+  namespaces:
+    - "*"
+  allowedServices: ['backend-.*']
+----
+
+NOTE: When exposing services, use the `--address <name>` parameter of the `skupper` CLI to name services appropriately.
+
+
+// Type: procedure
+[id="allowedExposedResources"] 
+=== Implement a policy to allow specific resources
+
+Use `allowedExposedResources` to specify which service bindings can be created or used on the {service-network}.
```

Comment:
Is this really 'allowed or used'?  I had the impression that restriction was for services only.  For exposed resources, the restriction would only be on the local resources being exposed, but that resources elsewhere that were on the VAN would not be affected by that.  @fgiorgetti, if you can confirm.

Reply:
This is used to determine which targets are allowed to be bound by an authorized service.
Ideally it must be expressed as: `*` or `<target-type>/<target-name>`, example:
`deployment/my-app`.